spack-environments/archer2/compute-node/spack.yaml

```diff
@@ -0,0 +1,371 @@
+        cxxflags: -Wno-unused-command-line-argument -mllvm -eliminate-similar-expr=false
+        fflags: -Wno-unused-command-line-argument -mllvm -eliminate-similar-expr=false
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-aocc
+      - aocc/3.0.0
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: aocc@3.2.0
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags:
+        cflags: null
+        cxxflags: null
+        fflags: null
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-aocc
+      - aocc/3.2.0
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: aocc@8.0.0
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags:
+        cflags: null
+        cxxflags: null
+        fflags: null
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-aocc
+      - aocc/8.0.0
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: aocc@8.1.0
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags:
+        cflags: null
+        cxxflags: null
+        fflags: null
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-aocc
+      - aocc/8.1.0
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: aocc@8.3.3
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags:
+        cflags: null
+        cxxflags: null
+        fflags: null
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-aocc
+      - aocc/8.3.3
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: cce@11.0.4
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags: {}
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-cray
+      - cce/11.0.4
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: cce@12.0.3
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags: {}
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-cray
+      - cce/12.0.3
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: cce@13.0.2
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags: {}
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-cray
+      - cce/13.0.2
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: gcc@8.1.0
+      paths:
+        cc: cc
+        cxx: CC
+        f77: ftn
+        fc: ftn
+      flags: {}
+      operating_system: sles15
+      target: any
+      modules:
+      - PrgEnv-gnu
+      - gcc/8.1.0
+      environment: {}
+      extra_rpaths: []
```

Comment:
This compiler should not be included - GCC 8 is only installed to support CCE, it does not work as a standalone compiler environment